src/parsers/manifest/dash/indexes/base.ts

```diff
@@ -194,8 +208,17 @@ export default class BaseRepresentationIndex implements IRepresentationIndex {
    * Construct init Segment.
    * @returns {Object}
    */
-  getInitSegment() : ISegment {
-    return getInitSegment(this._index);
+  getInitSegment() : ISegment | null {
+    const initSegment = getInitSegment(this._index);
+    if (initSegment !== null && initSegment.range === undefined) {
+      const mightBeStaticContent = initSegment.indexRange === undefined;
+      if (initSegment.privateInfos === undefined) {
+        initSegment.privateInfos = {};
+      }
+      initSegment.privateInfos.shouldGuessInitRange = true;
+      initSegment.privateInfos.mightBeStaticContent = mightBeStaticContent;
```

Comment:
Isn't `mightBeStaticContent: true` but `shouldGuessInitRange: false` possible?

Maybe not, just asking